server/scratchy_server/app.py

```diff
@@ -11,8 +11,14 @@
 from scratchy_server.resources.userres import UserRes, AllUserRes
 from scratchy_server.resources.messageres import MessageRes, AllMessageRes
 
+import yaml
+import logging
+
+with open('logging.yaml', 'r') as logging_file:
+    config_logging = yaml.load(logging_file, Loader=yaml.FullLoader)
+
+logging.config.dictConfig(config_logging)
```

Comment:
Got an error here : 
```

    File "/home/benjamin/Documents/MDL/LPH/repo/scratchy-1/server/scratchy_server/app.py", line 20, in <module>

    logging.config.dictConfig(config_logging)

    AttributeError: module 'logging' has no attribute 'config'
```


When I test logging in a python shell I got this version : 
```
╰─$ python                 
Python 3.8.10 (default, Jun  2 2021, 10:49:15) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import logging
>>> logging.c
logging.captureWarnings(  logging.collections       logging.critical(         logging.currentframe(     
>>> logging.c
logging.captureWarnings(  logging.collections       logging.critical(         logging.currentframe(     
>>> logging.__version__
'0.5.1.2'
>>> logging.config
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'logging' has no attribute 'config'
```